requirements.txt

```diff
@@ -0,0 +1,72 @@
+# This file is @generated by PDM.
+# Please do not edit it manually.
+
+annoy==1.17.3
+arrow==1.3.0
+certifi==2024.8.30
+charset-normalizer==3.3.2
+click==8.1.7
+codecarbon==2.7.1
+colorama==0.4.6; platform_system == "Windows"
+contourpy==1.3.0
+copairs @ git+https://github.com/cytomining/copairs.git@880f22a551bd897896d148a0b07baa99d981c6a9
+cycler==0.12.1
+drjit==0.4.6
+fonttools==4.54.1
+greenlet==3.1.1; (platform_machine == "win32" or platform_machine == "WIN32" or platform_machine == "AMD64" or platform_machine == "amd64" or platform_machine == "x86_64" or platform_machine == "ppc64le" or platform_machine == "aarch64") and python_version < "3.13"
+idna==3.10
+joblib==1.4.2
+kaleido==0.2.1.post1
+kiwisolver==1.4.7
+llvmlite==0.43.0
+markdown-it-py==3.0.0
+matplotlib==3.9.2
```

Comment:
I don't see monai?

Reply:
Yeah, I am not sure why this did not automatically come up in the requirements.txt file

Reply:
It appears this PR is not installing cyto-dl or its dependencies.

Reply:
@fatwir I think you had cyto-dl and its dependencies included in a commit yesterday? Let's talk more in person